README.md

```diff
@@ -163,11 +165,26 @@ to run Playright tests locally. To test the documentation locally:
 1. Run the Docs webserver and test the documentation by running: `npm start`
 1. Build the docs and verify by running: `npm run build`
 
-### Jest tests
+### Jest Tests
```

Comment:
No, sentence casing is supposed to be used in UI copy.

Reply:
Can you please point me to where it says that in the styleguide? 

Btw I don't really care either way here. What I care about is that things are consistent, and it is documented somewhere how that consistency looks like.

Reply:
> When writing **user interface copy,** use sentence casing.

This talks about the UI.

Reply:
Just out of curiosity, are there separate styleguides for documentation vs app UI? If yes, what's the link to the style guide for documentation? The style guide I linked to refers to the "Microsoft Writing Style Guide", which is for documentation. 

In my experience, UI and documentation typically follow a common branding, color scheme, and language. Since apps often show documentation inline in their UI, the boundaries between docs and UI become more and more fluid in my opinion. I would therefore try to go with only one styleguide for everything if possible.